src/mitol/openedx/apps.py

```diff
@@ -0,0 +1,13 @@
+"""Openedx app AppConfig"""
+import os
+from django.apps import AppConfig
+
+
+class OpenedxApp(AppConfig):
```

Comment:
MITxOnline already has an app named `openedx`  so you might need to either add a transitional app as done in authentication or rename this class and its label

Reply:
yeah sure, 
I'll add a transitional app